backend/settings.py

```diff
@@ -15,6 +15,8 @@
 import sys
```

Comment:
The PYC files above should not be committed. I can't figure out how to comment on them directly.